planner/core/exhaust_physical_plans.go

```diff
@@ -844,6 +844,7 @@ func (la *LogicalAggregation) getHashAggs(prop *property.PhysicalProperty) []Phy
 		return nil
 	}
 	hashAggs := make([]PhysicalPlan, 0, len(wholeTaskTypes))
+	// TaskType is also a kind of property, as a result, you may get a cop/root task return.
```

Comment:
This comment is not suited to be put here. It's better to be put in the `PhysicalProperty` struct, which located in "planner/property/physical_property.go".